File: Java/Car.java
package Java;

public class Car {
    Integer id;
    String license;
    Account driver;//Objeto de tipo driver
    Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }
    
    void printDataCar(){
        System.out.println("license: " + license + " | Driver name: " + driver.name + " | Passenger: " + passenger);
    }
}

File: JS/Car.js
// function Car(license, driver) {
//     this.id;
//     this.license = license;
//     this.driver = driver;
//     this.passenger;
// }

// Car.prototype.printDataCar = function () {
//     console.log(this.driver);
//     console.log(this.driver.name);
//     console.log(this.driver.document);
// }

class Car {
    constructor(licence, driver){
        this.id;
        this.licence = licence;
        this.driver = driver;
        this.passenger;
    }

    printDataCar() {
        console.log(this.driver);
        // console.log(this.driver.name);
        // console.log(this.driver.document);
    }
}